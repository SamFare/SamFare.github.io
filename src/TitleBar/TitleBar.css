div div .TitleBarText {
  font-size: 40pt;
  font-family: "Courier New", Courier, monospace;
  opacity: 1.0;
  margin-right: 20%;
}

section div .TitleBarLayout {
  z-index: 1000;
  width: 100%;
  display:inline-block;
  float: none;
  margin-bottom: 0 !important;
  border-width: 0px;
  position: absolute;
  top: 0;
  left: 0;
  box-shadow: 0px 0px;
  background-color: rgba(0,0,0,0.0);
  background-image: linear-gradient(180deg,rgba(0,0,0,0.00) 0%,rgba(0,0,0,0.00) 100%);
}

.nav-item {
  display: inline-block;
}

.logoButton {
  height:32px;
  margin-top: 20%;
  padding: 0;
}