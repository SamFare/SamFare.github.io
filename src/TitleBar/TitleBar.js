import React from "react";
import { Navbar, NavItem, Nav } from "react-bootstrap";

import css from  "./TitleBar.css";


export default class TitleBar extends React.Component{
  render(){
    return (
      <Navbar className={css.TitleBarLayout}>
        <Navbar.Header className={css.TitleBarText}>
          SamFare.me
        </Navbar.Header>
        <Nav>
          <NavItem href="https://github.com/SamFare">
            <img className={css.logoButton} src="assets/GitHub-Mark-64px.png"/>
          </NavItem>
          <NavItem href="https://medium.com/@samuel.fare">
            <img className={css.logoButton} src="assets/Monogram.png"/>
          </NavItem>
        </Nav>
      </Navbar>
    );
  }
}
