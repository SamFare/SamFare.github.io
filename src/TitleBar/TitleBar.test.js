import React from "react";
import TitleBar from "./TitleBar.js";
import renderer from "react-test-renderer";

import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

let context = {};

describe("a title bar", () => {
  beforeEach(() => {
    context.titeBar = renderer.create(
      <TitleBar />
    );
  });

  it("it has the image that was passed in", () => {
    expect(context.titleBar).toMatchSnapshot();
  });
});