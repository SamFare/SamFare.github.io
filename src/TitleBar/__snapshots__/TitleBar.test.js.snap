// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a title bar it has the image that was passed in 1`] = `undefined`;
